### ISSUE TITLE
Django dropped support for django.utils.six

### ISSUE THREAD

**Issue by lennsa** [wagtail/wagtail-react-streamfield#55]
```
File "/home/lennart/Envs/wagtailTest/lib/python3.6/site-packages/wagtail_react_streamfield/monkey_patch.py", line 1, in <module>
    from django.utils.six import wraps
ModuleNotFoundError: No module named 'django.utils.six'
```
django >= 3 no longer supports django.utils.six

**Comment by colinappnovation** [comment]
@lennsa see pull request: https://github.com/wagtail/wagtail-react-streamfield/pull/56

**Comment by thibaudcolas** [comment]
This is now fixed and released as v1.3.6.
